From 6e39a0298b0064cbfc7af809184299465d6a44e1 Mon Sep 17 00:00:00 2001
From: Daniel Rosenberg <drosen@google.com>
Date: Wed, 19 May 2021 16:39:26 -0700
Subject: [PATCH 02/38] Add exfat utilities for mkfs and fsck

Bug: 171854784
Test: Builds, Large sdcards formated as exfat if kernel supports it
Change-Id: Ie56b17f55f5027fdf714002b4c307603809fb707
---
 shell_and_utilities/Android.bp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/shell_and_utilities/Android.bp b/shell_and_utilities/Android.bp
index b7d7490dc..97e8d8eaf 100644
--- a/shell_and_utilities/Android.bp
+++ b/shell_and_utilities/Android.bp
@@ -18,9 +18,11 @@ phony {
         "awk",
         "bc",
         "bzip2",
+        "fsck.exfat",
         "ldd",
         "logwrapper",
         "mini-keyctl",
+        "mkfs.exfat",
         "mkshrc",
         "newfs_msdos",
         "reboot",
-- 
2.34.1

